components/docs-chef-io/content/automate/monitoring-alerting-centralized-logging-integration.md

```diff
@@ -0,0 +1,50 @@
++++
+title = "Chef Automate HA integrations"
```

Comment:
@dishanktiwari2501 
Please update all the techincal blog links should be actual public facing URLs and not the internal sharepoint link, please replace in all the 4 sections